common/app/model/content.scala

```diff
@@ -359,7 +363,8 @@ object Article {
       ("lightboxImages", lightbox.javascriptConfig),
       ("hasMultipleVideosInPage", JsBoolean(content.hasMultipleVideosInPage)),
       ("isImmersive", JsBoolean(content.isImmersive)),
-      ("isSensitive", JsBoolean(fields.sensitive.getOrElse(false)))
+      ("isSensitive", JsBoolean(fields.sensitive.getOrElse(false))),
+      ("videoDuration" -> videoDuration)
```

Comment:
will this conflict with the existing one here? https://github.com/guardian/frontend/blob/1cd351a59b6c4cfd5f50bc3ac267e84cef7c40b1/common/app/model/content.scala#L524